src/main/resources/db/migration/V1_5__create_courthouse_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE moj_courthouse (
+  moj_crt_id                INTEGER                         GENERATED ALWAYS AS IDENTITY
+, courthouse_name           CHARACTER VARYING               NOT NULL
+, code                      SMALLINT
+, created_date_time         TIMESTAMP with TIME ZONE     NOT NULL
```

Comment:
Hopefully we are going to get Barry to add these extra columns throughout?  I know he added some for Transcription recently. I think you're probably right having `NOT NULL` but it's not yet in our proposed design - so we may need to inform Barry as we encounter any discrepancy during dev.

I think for those 2 columns it's worth aligning with what Barry is using throughout.
`,created_ts                 TIMESTAMP WITH TIME ZONE               
,last_modified_ts           TIMESTAMP WITH TIME ZONE`

We will probably be going over them all to add another column for the user FK
`,last_modified_by           INTEGER`

Reply:
ACCEPT will change names to add ts but I think we need to wait for more discussion for last modified by column.